### DISCUSSION TITLE
Will I be able to sequentially select all tables in the document and assign new styles to them?

### DISCUSSION THREAD

**Discussion by docopshabr** [ONLYOFFICE/DocumentBuilder#138] [Q&A] [answered]
Right now I can't even select the first table. What method should I use for this?

**Comment by trofim24** [comment, marked as answer]
You can use `GetAllTables` [method](https://api.onlyoffice.com/docs/office-api/usage-api/text-document-api/ApiDocument/Methods/GetAllTables/) or `GetAllTablesOnPage` [method](https://api.onlyoffice.com/docs/office-api/usage-api/text-document-api/ApiDocument/Methods/GetAllTables/) and change style using `SetStyle` [method](https://api.onlyoffice.com/docs/office-api/usage-api/text-document-api/ApiTable/Methods/SetStyle/).
